corehq/warehouse/transforms/sql/app_status_synclog_staging.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO {{ app_status_synclog_staging }} (
+       last_sync,
+       sync_build_version,
+       user_id,
+       domain,
+       batch_id
+)
+SELECT
+    last_sync,
+    sync_build_version,
+    user_id,
+    domain,
+    '{{ batch_id }}'
+FROM
+(
+SELECT
+	row_number() over wnd,
+	CASE
+	    WHEN sync_date < app_status.last_sync_log_date
+	    THEN app_status.last_sync_log_date
+	    ELSE first_value(sync_date AT TIME ZONE 'UTC') OVER wnd
```

Comment:
it looks like this will get the oldest sync date. is that's what's desired? (just asking because it goes into a column called "last_sync")

Reply:
sorry nevermind. i see it's ordering descending